src/sqlfluff/core/rules/analysis/select.py

```diff
@@ -114,31 +114,40 @@ def get_aliases_from_select(segment, dialect=None):
     table_aliases = []
     standalone_aliases = _get_pivot_table_columns(segment, dialect)
     for table_expr, alias_info in aliases:
-        if _has_value_table_function(table_expr, dialect):
+        if has_value_table_function(table_expr, dialect):
             if alias_info[0] not in standalone_aliases:
                 standalone_aliases.append(alias_info[0])
-        elif alias_info not in standalone_aliases:
+        elif alias_info not in table_aliases:
```

Comment:
This is unrelated to the overall fix. It looks like a typo -- the intent here was to avoid adding duplicate entries to `table_aliases`.